package/MDAnalysis/coordinates/FHIAIMS.py

```diff
@@ -0,0 +1,261 @@
+strings for writing lines in ``.gro`` files.  These are as follows:
+
+``xyz``
+  An atom line without velocities.  Requires that the 'name' and 'pos' keys be supplied.
+  E.g.::
+
+     fmt['xyz'].format(pos=(0.0, 1.0, 2.0), name='O')
+
+``box_orthorhombic``
+  The final line of the gro file which gives box dimensions.  Requires
+  the box keyword to be given, which should be the three cartesian dimensions.
+  E.g.::
+
+     fmt['box_orthorhombic'].format(box=(10.0, 10.0, 10.0))
+
+``box_triclinic``
+  The (optional) initial lines of the file which gives box dimensions. 
+  Requires the box keyword, as a length 9 vector.  This is a flattened
+  version of the (3,3) triclinic vector representation of the unit
+  cell.  
+
+.. _FHIAIMS: https://aimsclub.fhi-berlin.mpg.de/
+
+"""
+from __future__ import absolute_import
+
+import re
+from six.moves import range, zip
+from six import raise_from
+
+import itertools
+import warnings
+
+import numpy as np
+
+from . import base
+from .core import triclinic_box, triclinic_vectors
+from ..exceptions import NoDataError
+from ..lib import util
+from ..lib import mdamath
+
+
+class Timestep(base.Timestep):
+
+    def _init_unitcell(self):
+        return np.zeros((3, 3), dtype=np.float32)
+
+    @property
+    def dimensions(self):
+        """unitcell dimensions (A, B, C, alpha, beta, gamma)
+        """
+        return triclinic_box(self._unitcell[0], self._unitcell[1], self._unitcell[2])
+
+    @dimensions.setter
+    def dimensions(self, new):
+        self._unitcell[:] = triclinic_vectors(new)
+
+
+class FHIAIMSReader(base.SingleFrameReaderBase):
+    """Reader for the FHIAIMS geometry format.
+
+    .. note::
+       Geometry files are single frame-only
+
+    """
+    format = ['IN', 'FHIAIMS']
+    units = {'time': None, 'length': 'Angstrom', 'velocity': None}
+    _Timestep = Timestep
+
+    def _read_first_frame(self):
+        with util.openany(self.filename, 'rt') as fhiaimsfile:
+            lines = fhiaimsfile.readlines()
+            names = np.asarray([line.split()[-1]
+                                for line in lines if 'atom' in line], dtype=object)
+            box = np.asarray([line.split()[1:] for line in lines[:4]
+                              if 'lattice' in line], dtype=np.float32)
+            positions = np.asarray(
+                [line.split()[1:-1] for line in lines if 'atom' in line], dtype=np.float32)
+            relative = [
+                'atom_frac' in line for line in lines if 'atom' in line]
+            positions[np.where(relative)] = np.matmul(
+                positions[np.where(relative)], box)
+            self.n_atoms = n_atoms = len(names)
+            self.ts = ts = self._Timestep(n_atoms, **self._ts_kwargs)
+            self.ts._unitcell[:] = box
+            ts._pos = positions.copy()
+            # No velocities in FHIAIMS geometry files
+            velocities = np.zeros((n_atoms, 3), dtype=np.float32)
```

Comment:
if there aren't velocities just don't make the array

Reply:
fixed